frontend/src/state/modules/layers/utils.js

```diff
@@ -1,4 +1,6 @@
 import { normalize } from 'normalizr';
+import { timeParse } from 'd3-time-format';
```

Comment:
The datepicker component uses `date-fns`. Are you using `d3-time-format` because the format used in the back-office is d3's?

If yes, could you update the user story to indicate this. We still have a TBD there.